src/plugins/jma/datasource.py

```
@@ -0,0 +1,383 @@
+    
+    async def get_metadata(self) -> Dict[str, Any]:
+        return {
+            'name': 'JMA',
+            'description': 'Japan Meteorological Agency - AMeDAS stations and JRA-55 reanalysis',
+            'resolution': 'Station data and 1.25x1.25 degrees reanalysis',
+            'temporal': 'Real-time AMeDAS, 6-hourly JRA-55',
+            'api_required': False,
+            'coverage': 'Japan'
+        }
+    
+    def validate(self, data: Dict[str, Any]) -> bool:
```

Ruff (0.12.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)